app/modules/intelligence/agents/chat_agents/pydantic_agent.py

```
@@ -322,10 +322,9 @@ def _create_multimodal_user_content(
 
                     # Test if base64 is valid
                     base64.b64decode(base64_data)
-                except Exception:
-                    logger.exception(
-                        f"Invalid base64 format for image {attachment_id}",
-                        attachment_id=attachment_id,
+                except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)